src/psyh/IsStringPalindrome.java

```diff
@@ -0,0 +1,13 @@
+package psyh;
+
+public class IsStringPalindrome {
+    public static boolean isStringPalindrome(String text) {
+        if (text == null) return false;
+        String textForAnalysis = text.toLowerCase().replaceAll("[^a-zA-Z0-9]", "");
+        StringBuilder reversOfTextForAnalysis = new StringBuilder();
+        for (int i = textForAnalysis.length() - 1; i >= 0; i--) {
```

Comment:
optional: you can make it more simple
        String reversOfTextForAnalysis = new StringBuilder(textForAnalysis).reverse().toString();
        return textForAnalysis.equals(reversOfTextForAnalysis);